transcom/mymove
# INT B-20070 PPM Address Instructions

## [B-20070](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20070)

## Summary

Updates some of the copy in `DateAndLocationForm.jsx` to consistently say that "you" will be moving the PPM rather than movers.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the customer app.
2. Login as a customer.
3. Begin creating a new PPM, this can be on an existing user or with existing orders.
4. On the very next screen you're shown after shipment type verify the following language is present: 
5. "What address are you moving from?" under the Origin header, 
6. "Please input your destination address." under the Destination header, 
7. "A second destination address could mean that your final incentive is lower than your estimate." under the form for the second destination address after selecting "Yes" to "Will you deliver part of your PPM to a different address?"


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
<img width="527" alt="Screenshot 2024-06-04 at 1 46 02 PM"
 src="https://github.com/transcom/mymove/assets/158500142/7e8e377b-2cd6-4d64-bc0e-db4ec76a2d19">
<img width="527" alt="Screenshot 2024-06-04 at 1 46 34 PM" src="https://github.com/transcom/mymove/assets/158500142/f3d89b58-cb08-4f99-86af-c763ea8ed501">
<img width="527" alt="Screenshot 2024-06-04 at 1 45 07 PM" src="https://github.com/transcom/mymove/assets/158500142/18acbfd0-9f9b-4817-8eea-1a3005f8df58">
